lib/plugins/print/print.test.js

```diff
@@ -326,38 +326,6 @@ describe('Print', () => {
     });
   });
 
-  it('should resolve self references', () => {
-    const conf = {
-      custom: {
-        me: '${self:}',
-      },
-      provider: {},
-    };
-    getServerlessConfigFileStub.resolves(conf);
-
-    serverless.processedInput = {
-      commands: ['print'],
-      options: {},
-    };
-
-    const expected = {
-      custom: {
-        me: {
-          $ref: '$',
-        },
-      },
-      provider: {},
-    };
-
-    return print.print().then(() => {
-      const message = print.serverless.cli.consoleLog.args.join();
-
-      expect(getServerlessConfigFileStub.calledOnce).to.equal(true);
-      expect(print.serverless.cli.consoleLog.called).to.be.equal(true);
-      expect(YAML.load(message)).to.eql(expected);
-    });
-  });
```

Comment:
I removed that test: it tested that the `${self:}` syntax worked. I had a look at the git history and the rest of the project but couldn't understand why this was tested.

Either I missed something and we should bring that back + cover it with the new regex (let me know), or it's something that isn't necessary anymore and we can move forward.